frontend/tests/components/auth/Login.test.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render, screen, waitFor } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import axios from 'axios';
+import userEvent from '@testing-library/user-event'
+import Login from '../../../src/components/auth/Login';
+import { AUTH_BASE_URI } from '../../../src/constants/api';
+
+
+jest.mock('axios');
+
+describe('Login component', () => {
+
+  // mock login request
+  axios.post.mockImplementation({ data: { 
+      expiry: "2023-02-09T14:52:04.877168Z",
```

Comment:
Is post response required by test?

Reply:
No, the response is not used in the test, in fact it could be removed. The only reason to maintain the mocked response, it's we have an example for future tests inside the projects.